scripts/generators/generate-package-manifest.js

```diff
@@ -1,37 +1,77 @@
 /**
  * This is used solely for V8 release.
- * @see {@link ../../azure-pipelines.release.yml}
+ * {@link file://./../../azure-pipelines.release.yml}
  */
 
+const fs = require('fs');
 const path = require('path');
-const fs = require('fs-extra');
+
+const { getAllPackageInfo, workspaceRoot } = require('@fluentui/scripts-monorepo');
 const semver = require('semver');
-const { getAllPackageInfo, findGitRoot } = require('@fluentui/scripts-monorepo');
 
-// Generate "manifest" file with package.jsons of all the monorepo packages (mainly for ODSP)
+main('v8', path.join(workspaceRoot, 'package-manifest'), '@fluentui/react');
+
+/**
+ * Generate "manifest" JSON file with map of package.json of monorepo packages that belong under provided tag group
+ * @remarks This is mainly/only for ODSP
+ * @param {string} tag - based on this only project containing this tag will be processed
+ * @param {string} manifestRoot - directory path where it should be generated
+ * @param {string} rootPackage
+ */
+function main(tag, manifestRoot, rootPackage) {
+  const allPackageInfo = getAllPackageInfo();
+
+  const filteredPackages = Object.entries(allPackageInfo).reduce((acc, curr) => {
```

Comment:
nit: might be easier to call `Object.fromEntries` after filtering by the object entries

Reply:
can you elaborate a bit more ? 

You mean using following ? 

`Object.fromEntries(Object.entries(allPackageInfo).map(...))` ? 

if that's the case I dont think its more readable/faster than reduce